### DISCUSSION TITLE
Testing Matter protocol Tasmota device with Google Home

### DISCUSSION THREAD

**Discussion by lboue** [arendst/Tasmota#18363] [General]
### Building Tasmota with Matter protocol Tasmota support
To be completed

### [Google Home Pairing restrictions ](https://developers.home.google.com/matter/integration/pair#pairing_restrictions)

A Matter device can only be paired in the Google Home ecosystem in certain Vendor ID and device type scenarios.

- A test VID cannot be used in a consumer device.
- For a production (consumer) device to be paired, it must be certified in the CSA DCL.
- For development and field trial purposes, a device must be registered in the Google Home Developer Console.
- You will need to install [GHSAFM-1.4.1-targetcommissioner-debug.apk](https://developers.home.google.com/samples/matter-app#commission_devices) on your Android smartphone

**Comment by s-hadinger** [comment]
Thanks. It would be better to post changes to the skeleton of documentation already in place. 

https://tasmota.github.io/docs/Matter/

**Reply by lboue** [reply]
Good idea.
